### PR TITLE
Updated ffmpeg channel to fix seg fault

### PR DESCRIPTION
**Pull request by prabhat00155**
Resolves https://github.com/pytorch/vision/issues/4430.
ffmpeg 4.3(latest) from pytorch channel causes seg fault. This PR replaces pytorch channel with conda-forge which works fine.

### CODE REVIEW

**Comment by NicolasHug** [comment]
I could be wrong but  I believe the reason we don't rely on the conda-forge channel is because this would defacto make torchvision's binaries gpl-licensed.

The ffmpeg in the pytorch channel is stripped down from the GPL-licensed code.

**Comment by prabhat00155** [comment]
> I could be wrong but I believe the reason we don't rely on the conda-forge channel is because this would defacto make torchvision's binaries gpl-licensed.
> 
> The ffmpeg in the pytorch channel is stripped down from the GPL-licensed code.

Does that mean we can't use it for CI? Sorry, I don't know much about how these licenses work.

**Comment by NicolasHug** [comment]
> Does that mean we can't use it for CI?

We technically can use it, as in, there's no licensing issue there. The licensing issue is about what ffmpeg version we can link against in our binaries: we can't link against conda-forge.
So testing against conda-forge wouldn't give us the signal we're looking for: we should test with the ffmpeg version that our binaries will be linked against.

**Comment by NicolasHug** [comment]
> we should test with the ffmpeg version that our binaries will be linked against.

Maybe my comment doesn't make much sense since the actual ffmpeg code that's eventually executed can be swapped.

But this makes me wonder why we even have a custom ffmpeg in the pytorch channel then 🤔
Sorry for the noise @prabhat00155 , perhaps @fmassa can provide more actionable feedback here

**Comment by andfoy** [comment]
Also, FFmpeg 4.3 has some errors that are in a assembly section of their code, so it is not a good idea to do that. After a discussion that we had earlier this month, it was decided that the FFmpeg "linking" symbols will be provided via a proxy package that is GPL and links statically against GPL-flavored FFmpeg, so rendering our own FFmpeg outdated.

**Comment by prabhat00155** [comment]
It still installs ffmpeg from pytorch channel:
```
(base) prabhatroy-mbp:notebooks prabhatroy$ ssh -p 64538 34.228.52.20
(base) [root@334f751efa90 ~]# conda activate /root/project/env
(/root/project/env) [root@334f751efa90 ~]# conda list | grep ffmpeg
ffmpeg                    4.3                  hf484d3e_0    pytorch
(/root/project/env) [root@334f751efa90 ~]# 

```

**Comment by bjuncek** [comment]
Just my two cents here to get @prabhat00155 unblocked:

1. At the moment we shouldn't (and don't IIUC) link against any FFMPEG at the moment; the FFMPEG from `pytorch` channel causes segfaults, and the `conda-forge` version of 4.3 (4.3.2, _NOT_ the 4.3.1 that segfaults) has been stable so far. 

2. @NicolasHug  at the moment we don't (and shouldn't) link FFMPEG binaries until we verify that the proposal @andfoy mentions (IIRC Nikita said he will write a prototype) works. 
So for now, our default is saying to the customers: "please install your own ffmpeg (however licensed you want) and rebuild from source. This _needs_ to be changed, and then we can update CI's accordingly.

**tl;dr** enabling this only makes sure that our tests are running with the version of FFMPEG that most people will be using, regardless of the licensing. When we have a better solution, we can reflect that in CI as well.

**Review by bjuncek — APPROVED** [review]
Approving to unblock @prabhat00155 and further testing; should be revisited once we find a way to link against correct FFMPEG
